setup.py

```diff
@@ -12,6 +12,7 @@
     "f90nml>=1.1.0",
     "numpy",
     "xarray",
+    "types-pyyaml",
```

Comment:
Not needed here if you put it in the pre-commit additional dependencies

Reply:
Removed.